README.md

```diff
@@ -35,56 +35,23 @@ The key features of the Intelligence Layer are:
 # Installation
 
 ## Local installation (for development and tutorials)
-Clone the Intelligence Layer repository from github.
-```bash
-git clone git@github.com:Aleph-Alpha/intelligence-layer.git
-```
-The Intelligence Layer uses `poetry` as a package manager. Follow the [official instructions](https://python-poetry.org/docs/#installation) to install it.
-Afterwards, simply run `poetry install` to install all dependencies in a virtual environment.
-```bash
-poetry install
-```
-The environment can be activated via `poetry shell`. See the official poetry documentation for more information.
-
-
-### Getting started with the Jupyter Notebooks
-
-After running the local installation steps, you can set whether you are using the Aleph-Alpha API or an on-prem setup via the environment variables.
-
----
-**Using the Aleph-Alpha API** \
-  \
-In the Intelligence Layer the Aleph-Alpha API (`https://api.aleph-alpha.com`) is set as default host URL. However, you will need an [Aleph Alpha access token](https://docs.aleph-alpha.com/docs/account/#create-a-new-token) to run the examples.
-Set your access token with
-
-```bash
-export AA_TOKEN=<YOUR TOKEN HERE>
-```
 
----
-
-**Using an on-prem setup** \
-  \
-In case you want to use an on-prem endpoint you will have to change the host URL by setting the `CLIENT_URL` environment variable:
+Clone the Intelligence Layer repository from github.
 
 ```bash
-export CLIENT_URL=<YOUR_ENDPOINT_URL_HERE>
+git clone git@github.com:Aleph-Alpha/intelligence-layer.git
 ```
 
-The program will warn you in case no `CLIENT_URL` is set explicitly set.
-
----
-After correctly setting up the environment variables you can run the jupyter notebooks.
-For this, run `jupyter lab` inside the virtual environment and go to the [examples](http://localhost:8888/lab/workspaces/auto-C/tree/src/documentation) directory.
```

Comment:
This one and the line above are now missing in your section. For people that don't have an integration in their IDE and are completely new to jupyter notebooks, this information is quite helpful.

So please add these lines :)